ec2rlcore/awshelpers.py

```diff
@@ -47,6 +47,18 @@ def get_instance_region():
     """
     try:
         r = requests.get("http://169.254.169.254/latest/dynamic/instance-identity/document")
+        if r.status_code == 401:
```

Comment:
If we end up with any places where we need to use IMDSv2, we should extract this handling to a generic function to reduce duplicate code.

Reply:
I don't think we're likely to need it in additional places, unless we start adding new functionality.

It would be fairly easy to rip out the token call into its own function, though. If we end up needing to use it someplace else, I'll go back and refactor this.